sdk/communication/communication-email/review/communication-email.api.md

```diff
@@ -0,0 +1,89 @@
+## API Report File for "@azure/communication-email"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { KeyCredential } from '@azure/core-auth';
+import { PipelineOptions } from '@azure/core-http';
+
+// @public
+export interface EmailAddress {
+    displayName?: string;
+    email: string;
+}
+
+// @public
+export interface EmailAttachment {
+    attachmentType: EmailAttachmentType;
+    contentBytesBase64: string;
+    name: string;
+}
+
+// @public
+export type EmailAttachmentType = "avi" | "bmp" | "doc" | "docm" | "docx" | "gif" | "jpeg" | "mp3" | "one" | "pdf" | "png" | "ppsm" | "ppsx" | "ppt" | "pptm" | "pptx" | "pub" | "rpmsg" | "rtf" | "tif" | "txt" | "vsd" | "wav" | "wma" | "xls" | "xlsb" | "xlsm" | "xlsx";
+
+// @public
+export class EmailClient {
+    constructor(connectionString: string, options?: EmailClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, options?: EmailClientOptions);
+    getSendStatus(messageId: string): Promise<SendStatusResult>;
+    send(emailMessage: EmailMessage): Promise<SendEmailResult>;
```

Comment:
These 2 seem to be missing `OperationOptions` parameter

Reply:
We didn't want to expose these options for these two methods in the SDK. I'll update the comment in emailClient.ts to remove this.